rfd/0036/README.md

```diff
@@ -67,23 +118,141 @@ These types may include:
+
+```
+                          +-------------+
+                      +---+Organizations| --+
+                      |   +-------------+   |
+                      |                     |
+                      |                     |
+                  +---v----+             +--v--+
+                  |Projects|<----------->|Users|
+                  +---+----+             +-----+
+                      |
+    +-----------+-----+-----+-----------+
+    |           |           |           |
++---v----+  +---v----+  +---v---+  +----v----+
+|Services|  |Networks|  |Storage|  |Unmanaged|
++--------+  +--------+  +-------+  | Compute |
+    |                              +---------+
+    |
++---v---+
+|Compute|
++-------+
+```
+
+In the above diagram "unmanaged compute" describes both existing instances that were defined before the introduction of services, as well as new instances that a user may define without first defining a service. Support for existing unmanaged compute and their ongoing use, as well as the ability to provision new unmanaged instances is required, despite the introduction of services. However, there is no requirement nor intention of providing a migration plan to convert a collection of existing unmanaged instances into a service.
 
-Scaling, upgrading, even stopping all the instances of an app can take time...sometimes significant time. To represent this to the user, Triton must expose the task queue and offer the ability to cancel jobs.
+Projects are described by a [project manifest](./projects/manifest.md), a YAML-formatted file that can be easily copied from elsewhere and in which changes are easily discernible in a text diff. They also have attached metadata as described below.
+
+[Read more about what projects mean in the Mariposa context](./projects), including Triton CLI commands and the manifest file.
 
-This document defines tasks specific to deploying services, but the task queue is not limited to services.
 
 ### Project meta (including secrets)
 
 Many applications require configuration values which are undesirable or unsafe to set in the application image. These can include license keys, a flag setting whether it's a staging or production environment, usernames and passwords, and other details.
 
 This document proposes a simple method of storing those details and injecting them into containers. It is not intended to provide the rich features of solutions like Hashicorp's Vault, instead it is intended to provide a basic solution that is easy to use in a broad variety of applications.
+
+[Read more about meta in the Mariposa context](./meta), including Triton CLI commands and the manifest file.
```

Comment:
>It is not intended to provide the rich features of solutions like Hashicorp's Vault, instead it is intended to provide a basic solution that is easy to use in a broad variety of applications.

I'm not sure it is appropriate to hand wave over the details here. From my perspective this reads a lot like "Vault is difficult, ours will work, but it's not all that secure".